unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
```

Comment:
Hmmm... if we can do react views with NONE of the react state management, and dynamic-updating, then yes. But otherwise i'd rather do something trivially simple, and then later teach them react. Who knows where they'll go to work after this, and they should know a simple server side templating system from a super powerful client side one.